### PR TITLE
Adding GetWorkflowExecutionHistory grpc method #1

### PR DESCRIPTION
**Pull request by noiarek**
**What changed?**
Adding new grpc method (GetWorkflowExecutionHistory)

**Why?**
Part of the GRPC migration

**How did you test it?**
Unit Test

**Potential risks**
None

 
**Release notes**

### CODE REVIEW

File: cadence/cadence_types.py [hunk 1]
```diff
@@ -295,6 +295,7 @@ def value_for(cls, n: int) -> PendingActivityState:
 class HistoryEventFilterType(IntEnum):
     ALL_EVENT = 0
     CLOSE_EVENT = 1
+    INVALID = 2
```
Comment (longquanzheng) [inline cadence/cadence_types.py:298]: why is this needed? Probably for tests?

**Review by longquanzheng — APPROVED** [review]

File: cadence/cadence_types.py (hunk 1 above)
Comment (noiarek) [reply cadence/cadence_types.py:298]: All the enums on grpc now have a `Invalid` value = 0. I'm adding it to all our instances
